cpu/stm32f1/lpm_arch.c

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright (C) 2013 INRIA
- * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
I didn't write a single line of code on this file :-) 
Do we have a policy on when to add a Copyright/Author and when not?